Major Constitution Revision (#16)
ritsec/constitution: constitution/article-07.tex

\Section{A-Board Status}
\begin{enumerate}
  \item Members elected to be A-Board retain all standard Member rights, unless otherwise specified. 
  \item Each individual on an A-Board position must directly report to those described in their respective sections.

>   \item The creation of A-Board position is at the discretion of the E-Board.

Reviewer: ```suggestion
  \item The creation of an A-Board position is at the discretion of the E-Board.
```

\begin{enumerate}
  \item The interview process for this position is up to the discretion of the incoming E-Board. 
  \item This position is appointed by a vote of the incoming E-Board.

>   \item This position directly reports to Vice President. 

Reviewer: ```suggestion
  \item This position directly reports to the Vice President. 
```

\begin{enumerate}
  \item The interview process for this position is up to the discretion of the current Tech Lead.
  \item This position is appointed by a vote of the incoming E-Board.
  \item This position directly reports to and is responsible for assisting the current Tech Lead.
  \item This individual is unable to participate in the demos or obtain demo prizes.
\end{enumerate}

% SECTION C: HEAD OF SOCIAL MEDIA
\Section{Head of Social Media} 

\begin{enumerate}
  \item The interview process for this position is up to the discretion of the current Head of Public Relations.

>   \item This position is appointed by a vote of incoming E-Board.

Reviewer: ```suggestion
  \item This position is appointed by a vote of the incoming E-Board.
```

>   \item This position directly reports to Head of Public Relations.

Reviewer: ```suggestion
  \item This position directly reports to the Head of Public Relations.
```

>   \item Responsible for maintaining all RITSEC Social Media. 

Reviewer: ```suggestion
  \item Responsible for maintaining all of RITSEC's Social Media. 
```
